projects/bazel-rules-fuzzing-test/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -eu
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+declare -r QUERY='
+    let all_fuzz_tests = attr(tags, "fuzz-test", "//...") in
+    $all_fuzz_tests - attr(tags, "no-oss-fuzz", $all_fuzz_tests)
+'
+declare -r OSS_FUZZ_TESTS="$(bazel query "${QUERY}" | sed 's/$/_oss_fuzz/')"
```

Comment:
maybe just '_fuzzer' suffix, we don't create targets with _oss_fuzz suffix

Reply:
This is actually a generated target. Each fuzz target `<name>` gets a `<name>_oss_fuzz` target that produces the OSS-Fuzz artifacts that can be copied in the `${OUT}/` directory.

Do you have a preference for calling this differently?